common.yml

```diff
@@ -439,7 +168,7 @@ functions:
         if [ "${USE_TLS}" = "true" ]; then
           MONGOD_ARGS="${mongod_args_tls}"
         elif [ "${AWS_AUTH}" = "true" ]; then
-          MONGOD_ARGS="--auth --setParameter authenticationMechanisms=MONGODB-AWS,SCRAM-SHA-256"
```

Comment:
what's the reason for this removal?

Reply:
As far as I can tell it isn't a valid option for `authenticationMechanisms`. It was causing system failures for the `aws-auth` test.